crates/project_panel/src/project_panel.rs

```diff
@@ -2982,8 +2982,9 @@ impl ProjectPanel {
                 .right_1()
                 .top_1()
                 .bottom_1()
-                .w(px(12.))
```

Comment:
I changed the width because otherwise there would be a gap between the right side of the scroll handle and where the blue border continues:
![grafik](https://github.com/user-attachments/assets/8119ddaf-c3db-466b-9809-a81f60c5dc23)
In my opinion it looks better like this:
![grafik](https://github.com/user-attachments/assets/46d48b02-8cb7-4a8a-94b4-17d31315ca42)
but i think it's a matter of taste

Reply:
Maybe this could be solved in a different way, but i don't know how, this is my first contribution to this project